keyboards/jian/rev2/readme.md

```diff
@@ -0,0 +1 @@
+This is GB revision
```

Comment:
I feel like `keyboards/jian/readme.md` should call out that the rev2 is the GB revision.